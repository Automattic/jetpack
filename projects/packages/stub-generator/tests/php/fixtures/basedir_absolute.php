<?php

return array(
	'basedir' => __DIR__ . '/files',
	'files' => array(
		'functions.php' => '*',
	),
);
