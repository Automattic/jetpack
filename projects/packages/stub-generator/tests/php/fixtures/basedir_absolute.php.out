<?php
/**
 * A function.
 *
 * @param string $p1 Parameter 1.
 * @param array $p2 Parameter 2.
 * @return ?object Some object.
 */
function a_function($p1, $p2)
{
}
/**
 * Another function.
 *
 * @return never
 * @throws Exception Always.
 */
function another_function()
{
}
function undocumented_function($args)
{
}
